python/Framework.py

```diff
@@ -77,10 +77,10 @@ def __init__(self, options, verbose=True):
                 input_files = cms.vstring()
                 )
 
-        process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_condDBv2_cff')
-        process.load('Configuration.StandardSequences.GeometryRecoDB_cff')
-        process.load('Configuration.StandardSequences.MagneticField_38T_cff')
         process.load('FWCore.MessageLogger.MessageLogger_cfi')
+        process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
+        process.load('Configuration.StandardSequences.MagneticField_cff')
+        process.load('Configuration.Geometry.GeometryExtended2023D17Reco_cff')
```

Comment:
Forgive my ignorance, but can't you get the geometry from the database (using the global tag)?

Reply:
When I tried without the GeometryExtended2023D17Reco_cff, I got a segmentation fault in the package computing HGC electron identification.